src/app/appmarket/domains/list/domainslist.component.html

```diff
@@ -84,4 +84,6 @@ <h3>{{ 'DOMAINS.TITLE' | translate }}</h3>
 
 </div>
 
-<app-removal-confirmation-modal [domain]="domainToRemove" (onConfirm)="softRemoveDomain(domainToRemove.id)"></app-removal-confirmation-modal>
+<app-removal-confirmation-modal [object]="domainToRemove" [disableButton]="false" (onConfirm)="softRemoveDomain(domainToRemove.id)"
```

Comment:
Could we rename this modal? It contains `app` in the name but I suppose it's used in few places

Reply:
Or maybe the `app` should be replaced with `domain` in this case